docs/editor/accessibility.md

```diff
@@ -25,6 +25,46 @@ You can adjust the Zoom level in VS Code with the **View** > **Appearance** > **
 
 When you adjust the zoom level with the **View** > **Zoom In / Out** commands, the zoom level is persisted in the `window.zoomLevel` [setting](/docs/getstarted/settings.md). The default value is 0 and each increment/decrement changes the zoom level by 20%.
 
+## Color vision accessibility
+
+You can search for extensions in **Visual Studio Marketplace** that are compatible for color vision deficiency. Use Extensions `kbstyle(Ctrl+Shift+X)` to access the Marketplace extension search bar and enter "colorblind" to populate relevant options.
+
+![Visual Studio Marketplace in VS Code UI](images/accessibility/accessibility-extension-marketplace.png)
+
+To change the [color theme](/docs/getstarted/themes.md), go to **File** > **Preferences** > **Color Theme** `kbstyle(Ctrl+K)` `kbstyle(Ctrl+T)`.
```

Comment:
For the Preferences menu, it is File > Preferences > Color Theme on Windows and Linux but Code > Preferences > Color Theme on macOS.
We often say something like "go to **File** > **Preferences** > **Color Theme** (**Code** > **Preferences** > **Color Theme** on macOS)"
and you would use `kb(workbench.action.selectTheme)` instead of kbstyle

Reply:
Looks like this is a general issue for many places in the topic, can you update the other places as well.
I wonder if it might be sufficient to call it out with the parenthetic text once per section and then just go with **File/Code** > **Preferences**?